i18n/zh/docusaurus-plugin-content-blog/2022-10-26-test-report/index.md

```diff
@@ -1,10 +1,21 @@
 ---
-title: 突破100集群和500,000节点，Karmada大规模测试报告发布
+title: 突破100倍集群规模，Karmada大规模测试报告发布
 ---
 
 ## 摘要
 
-随着Karmada在越来越多的企业和组织中的大规模落地，Karmada的可扩展性和大规模逐渐成为社区用户的新关注点。因此，我们对Karmada开展了大规模测试工作，根据测试结果分析，以Karmada为核心的集群联邦可以稳定支持100个集群和50万个节点同时在线，管理超过200万个Pod。在本文中，我们将介绍用于测试的相关指标，如何进行大规模测试，以及我们如何实现大规模节点和集群接入。
+随着云原生技术在越来越多的企业和组织中的大规模落地，如何高效、可靠地管理大规模资源池以应对不断增长的业务挑战成为了当下云原生技术的关键挑战。
+在过去的很长一段时间内，不同厂商尝试通过定制Kubernetes原生组件的方式扩展单集群的规模，这在扩大规模的同时也引入了复杂的单集群运维、不清晰的集群升级路径等问题。
+而多集群技术能在不侵入修改Kubernetes单集群的基础上横向扩展资源池的规模，在扩展资源池的同时降低了企业的运维管理等成本。
+
+在Karmada的大规模落地进程中，Karmada的可扩展性和大规模逐渐成为社区用户的新关注点。
+因此，我们对Karmada开展了大规模环境下的测试工作，以获取Karmada管理多个Kubernetes集群的性能基线指标。
+**对于以Karmada为代表的多集群系统而言，单集群的规模不是制约它的资源池规模的限制因素。**
```

Comment:
```suggestion

**对于以Karmada为代表的多集群系统而言，单集群的5k节点上限不再是制约资源池规模扩张的限制因素。**
```

Reply:
The scale here refers not only to nodes, but also to the number of Pods, the number of namespaces, and so on.